addons/godot-xr-tools/functions/movement_jump.gd

```diff
@@ -2,12 +2,13 @@ tool
 class_name XRToolsMovementJump
 extends XRToolsMovementProvider
 
+
 ##
 ## Movement Provider for Jumping
 ##
 ## @desc:
 ##     This script provides jumping mechanics for the player. This script works
-##     with the PlayerBody attached to the players ARVROrigin.
+##     with the player body attached to the players ARVROrigin.
```

Comment:
If we are referring to something that is a class, I think it makes sense to use the class name here, so `XRToolsPlayerBody` I guess?